README.md

```diff
@@ -510,14 +510,14 @@ metadata:
   ...
 spec:
   versions:             # []MappingTemplate
-  - version:              # string
-    annotations:          # string (Restricted JSONPath), optional
-    containers:           # []MappingContainer, optional
-    - path:                 # string (JSONPath)
-      name:                 # string (Restricted JSONPath), optional
-      env:                  # string (Restricted JSONPath), optional
-      volumeMounts:         # string (Restricted JSONPath), optional
-    volumes:              # string (Restricted JSONPath), optional
+  - version:            # string
```

Comment:
I find it easier to read when the comments follow the indentation of the lines they are commenting.

Reply:
But we are not consistent about the indentation. Can I change the whole document to follow a single style?
I am fine with the style Scott suggested.

Reply:
I changed the indentation to a single space: 16ce04c165963bed51bb8d5639489e1006f7f003